### ISSUE TITLE
Black-box Tests for both platforms

### ISSUE THREAD

**Issue by rofrischmann** [rofrischmann/react-look#234]
While unit-test coverage is almost 100% we yet do not test the whole resolving as used in real-life.
We should add black-box tests to cover real-life examples and to check if the whole ecosystem works together properly.

Could also be useful to extract some Test Utils for other devs such as `getElementStyles` which includes all className styles referenced.

**Comment by rofrischmann** [comment]
We might want to use [Enzym](https://github.com/airbnb/enzyme) which really looks create.

**Comment by alitaheri** [comment]
Enzyme is awesome indeed. But we don't have complex dom, but rather complex css output. But that depends on how you wish to blackbox test this library. :grin:

**Comment by rofrischmann** [comment]
Yeah, I guess it's a combination of both. I am currently preparing some test utils :p

Perhaps I can somehow integrate those with Enzym, did nit check the API right now.

**Comment by rofrischmann** [comment]
Test utils are there. Now we can start creating blackbox tests :)

**Comment by jordanh** [comment]
Bandwidth issues aside, is this the only issue preventing the release of 1.0.0?

**Comment by rofrischmann** [comment]
Heyhey @jordanh. Actually 1.0.0 is already out :P But yet we still should have better blackbox tests I guess. But it seems like I am not able to finish those too soon. Any help is appreciated as always. I am totally stressed out with my final thesis right now^^
